action.yml

```diff
@@ -17,6 +17,12 @@ inputs:
     description: 'Report level for reviewdog [info,warning,error]'
     default: 'error'
     required: false
+  name:
```

Comment:
How about `tool_name`?
The template of reviewdog actions uses it.
https://github.com/reviewdog/action-composite-template/blob/7abf5b1c4ee23110be4fa909cd413e3254ade14d/action.yml#L12-L14

Reply:
Changed to `tool_name`